src/providers/bedrock/utils.ts

```diff
@@ -8,6 +8,7 @@ import {
 } from './chatComplete';
 import { Context } from 'hono';
 import { env } from 'hono/adapter';
+import fs from 'fs/promises';
```

Comment:
Hey! Will `fs` imports work with workerd environment? Can we make it conditional based on runtime?

Reply:
done